hooks/boot/21-disable_system_checks.rb

```diff
@@ -0,0 +1,7 @@
+# Add reset option
+app_option(
+  '--disable_system_checks',
```

Comment:
Please use dashes not underscores so it's consistent with all other installer options